library/std/src/sys/unix/l4re.rs

```diff
@@ -481,7 +481,7 @@ pub mod net {
 
     impl LookupHost {
         pub fn port(&self) -> u16 {
-            unimpl!();
+            0 // unimplemented
```

Comment:
Does this code actually get reached in your programs? Keeping `unimpl!()` until it is actually implemented will make it very clear when you accidentally depend on it, instead of potentially getting confused by a 0 value being returned for some reason.

Reply:
No, this code only exists to provide a uniform interface on "unix" platforms. The problem is that `port()` returns a `u16`, so the `Err()` returned by `unimpl!()` can't be used here. While the mock implementation could follow the structure of other platforms without networking support more closely, returning `0` is more or less in line with what other platforms such as [sgx](https://github.com/rust-lang/rust/blob/595088d602049d821bf9a217f2d79aea40715208/library/std/src/sys/sgx/net.rs#L473) or [wasi](https://github.com/rust-lang/rust/blob/595088d602049d821bf9a217f2d79aea40715208/library/std/src/sys/wasi/net.rs#L350) do. Since the commit only fixes up the most pressing issues with the existing mainline implementation, my suggestion would be to leave the `0` for now.